src/lib/fetch.ts

```diff
@@ -4,20 +4,30 @@ import { URL } from 'url';
 import * as fs from 'fs';
 import { readFile } from 'fs/promises';
 
-export async function azleFetch(input: any, init?: any): Promise<any> {
+export async function azleFetch(
+    input: RequestInfo | URL,
+    init?: RequestInit | undefined
+): Promise<Response> {
     if (process.env.AZLE_TEST_FETCH === 'true') {
         console.log('azleFetch has been called');
     }
 
+    if (typeof input !== 'string') {
```

Comment:
If we add support for URL and Request object I would love to see some tests for it.

Reply:
We aren't going to add support for Request, but I will add tests for URL

Reply:
I've added one test for URL